msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalSdk.java

```diff
@@ -85,6 +89,17 @@ public MsalAuthResult acquireTokenInteractive(@NonNull MsalAuthTestParams authTe
             acquireTokenParametersBuilder.withClaims(authTestParams.getClaims());
         }
 
+        if (authTestParams.getAuthScheme() == AuthScheme.POP) {
+            acquireTokenParametersBuilder.withAuthenticationScheme(
+                    PoPAuthenticationScheme.builder()
+                            .withHttpMethod(HttpMethod.valueOf("GET"))
+                            .withClientClaims("")
```

Comment:
We can probably remove this line altogether I believe (for setting client claims). You can try removing and if nothing breaks then keep it removed.

Reply:
This feedback applies below to silent as well